blog/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  
+  def create
+    authenticate_user!
+    @article = articles_repo.find(params[:article_id])
+    @comment = comments_repo.new_entity(comment_params)
+    @comment.article_id = @article.id
```

Comment:
You don't need to do this.
It's enough to just save a comment record with proper article_id